client/web/src/components/Byline/CreatedByAndUpdatedByInfoByline.tsx

```diff
@@ -9,26 +9,43 @@ type UserData = Maybe<Pick<UserAreaUserFields, 'url' | 'username'>>
 
 interface BylineProps {
     createdAt: string
-    createdBy: UserData
+    createdBy?: UserData
     updatedAt: Maybe<string>
-    updatedBy: UserData
+    updatedBy?: UserData
+    noAuthor?: boolean
 }
 
 /**
  * The created/updated byline containing information about creator, creation date, updater and update date.
  */
-export const CreatedByAndUpdatedByInfoByline: FC<BylineProps> = ({ createdAt, createdBy, updatedAt, updatedBy }) => (
-    <>
-        Created <Timestamp date={createdAt} /> by{' '}
-        {createdBy ? <Link to={createdBy.url}>{createdBy.username}</Link> : 'a deleted user'}
-        {updatedAt !== null && updatedAt !== createdAt && (
-            <>
-                <span className="mx-2">|</span>
-                Updated <Timestamp date={updatedAt} />
-                {updatedBy?.username !== createdBy?.username && (
-                    <> by {updatedBy ? <Link to={updatedBy.url}>{updatedBy.username}</Link> : 'a deleted user'}</>
-                )}
-            </>
-        )}
-    </>
-)
+export const CreatedByAndUpdatedByInfoByline: FC<BylineProps> = ({
+    createdAt,
+    createdBy,
+    updatedAt,
+    updatedBy,
+    noAuthor,
+}) => {
+    const createdByPart = noAuthor ? (
+        ''
+    ) : (
+        <> by {createdBy ? <Link to={createdBy.url}>{createdBy.username}</Link> : 'a deleted user'}</>
+    )
```

Comment:
Not sure, but you might be able to do this, since you can use `null` in a JSX fragment:

```suggestion
    const createdByPart = noAuthor ??
        <> by {createdBy ? <Link to={createdBy.url}>{createdBy.username}</Link> : 'a deleted user'}</>
```